spec/features/admin/edit_organisation_details_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'features/support/sign_up_helpers'
+
+describe 'editing an organisations details' do
+  let!(:admin_user) { create(:user, super_admin: true, organisation: organisation) }
+  let!(:organisation) { create(:organisation, name: "TestMe & Company", service_email: "testme@gov.uk") }
+
+  context 'when visiting the organisations settings page before any changes' do
+    before do
+      sign_in_user admin_user
+      visit organisations_path(organisation)
+    end
+
+    it 'has the current compnay name and service email' do
+      expect(page).to have_content("Organisation Name: TestMe & Company")
+      expect(page).to have_content("Service Email: testme@gov.uk")
+    end
+  end
+
+  context 'when visiting the edit organisations settings page' do
+    before do
+      sign_in_user admin_user
+      visit edit_organisation_path(organisation)
+    end
+
+    it 'allows the user to change their company name and service email' do
+      fill_in 'org-name', with: "New Company Name"
```

Comment:
You should just be able to use the label, eg: `fill_in "Organisation name"` rather than having to create an arbitrary ID.

Reply:
This isn't working. when I run tests it can't find the element, why I changed it to an ID @tommotaylor